librenms/docker
# allow DB_PASSWORD with # character

## Problem

When using MYSQL_PASSWORD that contains `#` character, the entrypoint scripts write the `/opt/librenms/.env` file in a way that Laravel will interpret as a comment. Source: https://laravel.com/docs/5.8/upgrade#environment-variable-parsing

## Proposed solution

In the `03-config.sh` entrypoint file, wrap the DB_PASSWORD entry in the `/opt/librenms/.env` in double quotes. This allows Laravel (and underlying phpdotenv package) to handle the `#` correctly.

## Test steps

Even when quoting the value of `MYSQL_PASSWORD` in the docker compose `.env`, the value gets written to the `/opt/librenms/.env` file without quotes. You can confirm in `test/` environment on default branch:

1. cd `test`
1. Update `.env` with something like `MYSQL_PASSWORD="asupersecretpassword-test#hash"`
1. Start compose stack: `docker compose up -d`
1. View librenms web logs: `docker compose logs -f librenms`
1. On a new database, migration scripts will fail with incorrect password error
1. Verify that access to db container works with MYSQL_PASSWORD: `docker compose exec db mysql -u librenms -p`

Repeat test steps on this PR branch and database migrations should pass. Some other helpful commands to check DB_PASSWORD while in the librenms web container:
- `cat .env`
- `lnms tinker` -> in new prompt, enter `env("DB_PASSWORD")`
- `lnms db`
